server/statistics/hot_peer_cache.go

```diff
@@ -145,21 +142,30 @@ func (f *hotPeerCache) collectRegionMetrics(loads []float64, interval uint64) {
 	}
 }
 
+// CollectExpiredItems collects expired items, mark them as needDelete and puts them into inherit items
+func (f *hotPeerCache) CollectExpiredItems(region *core.RegionInfo) []*HotPeerStat {
```

Comment:
Since they are processed separately, is it possible that the old item is written to inherit itmes after the new peer stat has been processed?